src/App.js

```diff
@@ -1,5 +1,38 @@
+import { 
+  getComputerNumber, 
+  getGameResult, 
+  printGameResult 
+} from './utils/game';
+import { 
+  getInputNumber, 
+  getInputContinueNumber 
+} from './utils/user';
+import { MissionUtils } from '@woowacourse/mission-utils';
+import { GAME_MESSAGE } from './constants/gameMessage';
+
 class App {
-  async play() {}
+  async play() {
+    let computerNumber = await getComputerNumber();
```

Comment:
`getComputerNumber()`함수의 return 값을 저장하고 다시 재할당되지 않기 때문에
`const`로 선언해도 괜찮을 것 같아요!
```suggestion
    const computerNumber = await getComputerNumber();
```
관련하여 읽어보면 좋을 것 같은 포스트 하나 올려드립니다!
https://hyunseob.github.io/2016/11/21/misunderstanding-about-const/

Reply:
아래 보시면 게임 다시 시작할 때 재할당을 하고 있어서 let으로 선언했습니다..!!!
올려주신 글 너무 좋네요 꼭 읽어보겠슴니다 :)